### DISCUSSION TITLE
[PvP] First arena win of the day

### DISCUSSION THREAD

**Discussion by vvranceanu** [chromiecraft/chromiecraft#6916] [Suggestions]
Similar to a BG first win of the day, first arena win of the day will give people an incentive to log in and try to get into that arena que knowing other people will most likely try to do the same thing. As is the case on tuesday/weekends, once the arena que gets popping, most people will probably continue to play and we should have more arena play sessions throughtout the week. This will also create a habit for people and WoW was always all about creating habits for players to keep them hooked. That could get things going for more pvp people to join.

**Comment by balleny** [comment]
first win could also have the following negative impacts:

- more wintrading at odd times
- less playerbase satisfaction because of a "soft" force to play arena with the majority of bugs and imbalances present (esp. if it is linked to the first win, not the first game)

I would see this as an interesting option if there was a single player queue, where you get mixed into random 3vs3 teams, but I dislike this personally for regular 2s, 3s or 5s

**Reply by tbcxd** [reply]
Soft force to play bgs is also there with 10  ap first win
I strongly agree with OP and strongly disagree with you, lol.
Arenas are dead and need work because the rewards/time needed are not worth it for your average player to even consider playing it.

It's not the "possitive bugs and burst" that keeps people away from being active in arena, it's the fact that the rewards are just too time consuming.

**Reply by balleny** [reply]
taking the full season to get the full set surely isnt correct - but thats more a topic for arena start rating and arena point calculation
